src/core_plugins/console/public/src/input.js

```diff
@@ -178,8 +178,13 @@ export function initializeInput($el, $actionsEl, $copyAsCurlEl, output) {
 
             var warnings = xhr.getResponseHeader("warning");
             if (warnings) {
-              warnings = _.map(warnings.split(", "), function (warning) {
-                  return "#! Deprecation: " + warning;
+              // pattern for valid warning header
+              var re = /\d{3} [^ ]+ \"([^"]*)\"( \"[^"]*\")/
```

Comment:
That's a good question but I do not think it's necessary. The warning headers produced by Elasticsearch [will not contained escaped quotes](https://github.com/elastic/elasticsearch/pull/23275/files#diff-a2f8f23221b079e15cfc451728820f92R136).